Project.toml

```diff
@@ -46,7 +45,7 @@ DataStructures = "0.18"
 Distributions = "0.23.3, 0.24, 0.25"
 DistributionsAD = "0.6"
 DocStringExtensions = "0.8"
-DynamicPPL = "0.15"
+DynamicPPL = "0.15, 0.16"
```

Comment:
Possibly a stupid question, but shouldn't this be pinned to only `0.16` to avoid having to support two versions?

Reply:
It's not an issue as long as it's not a lie, i.e. we're actually compatible with both.

And the idea isn't to have to support two versions: if it turns out that 0.15 doesn't work anymore, then we just remove the compat entry.